.github/workflows/therock-ci-linux.yml

```
@@ -78,7 +78,7 @@ jobs:
       - name: Fetch sources
         timeout-minutes: 30
         run: |
-          ./TheRock/build_tools/fetch_sources.py --jobs 12 --no-include-math-libs
+          ./TheRock/build_tools/fetch_sources.py --jobs 12 --no-include-rocm-systems --no-include-rocm-libraries --no-include-ml-frameworks
```

This will need more work (ongoing in #1148) as the job matrix currently tries to build rocm-libraries components:

https://github.com/ROCm/rocm-systems/blob/3fd56bfefc1e08d40b2c48f7f630e3a0eb57cd17/.github/scripts/therock_matrix.py#L14
https://github.com/ROCm/rocm-systems/blob/3fd56bfefc1e08d40b2c48f7f630e3a0eb57cd17/.github/scripts/therock_matrix.py#L19

Limit your change to
```suggestion
          ./TheRock/build_tools/fetch_sources.py --jobs 12 --no-include-rocm-systems --no-include-ml-frameworks
```